base.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM registry.ci.openshift.org/ocp/builder:golang-1.15
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8
```

Comment:
Just to dive into this change some more:

- The `builder:golang-1.16` tag doesn't exist:

```bash
$ podman pull registry.ci.openshift.org/ocp/builder:golang-1.16
Trying to pull registry.ci.openshift.org/ocp/builder:golang-1.16...
  manifest unknown: manifest unknown
Error: Error initializing source docker://registry.ci.openshift.org/ocp/builder:golang-1.16: Error reading manifest golang-1.16 in registry.ci.openshift.org/ocp/builder: manifest unknown: manifest unknown
```